Makefrag

```diff
@@ -0,0 +1,43 @@
+# FLEXPRET_ROOT_DIR and NAME must be defined
```

Comment:
Can we check this and throw an error if not?

Reply:
yes, just added checks